### PR TITLE
#162163160 Configure Authors Haven to use PostgreSQL DB Engine 

### PR DESCRIPTION
**Pull request by sekayasin**
### What does this PR do?
Configure Authors Haven to use postgresql DB Engine
#### How should this be manually tested?
1. Make sure PostgreSQL is installed and create a database. Ref to Points in the Background context
2. Clone the repo && cd into ah-backend-zeus
- pip install -r requirements.txt
- python manage.py makemmigrations
- python manage.py migrate
- python manage.py createsuperuser 
- python manage.py runserver
#### Any background context you want to provide?
We opted to configure PostgreSQL as our DB engine and use it in production because: 
- PostgreSQL is the advanced, open-source [object]-relational database management system. 
- it is extremely capable of handling many tasks very efficiently. 
- It is highly programmable, and therefore extendible, with custom procedures that are called "stored procedures Compared to other RDBMSs, and mainly the SQLite db used by default which is a file based.

1. Postgresql MUST be installed locally, Create an empty database named "ah_zeus_db" with createdb utility or psql, create these environment variables in .env file
```
DB_NAME='ah_zeus_db'
DB_USER='username'
DB_HOST='127.0.0.1'
DB_PORT='5432'
```
- createdb ah_zeus_db -U username 
2. Using psql
- create database ah_zeus_db

#### What are the relevant pivotal tracker stories?
[#162163160](https://www.pivotaltracker.com/story/show/162163160l)

### CODE REVIEW

**Review by alexkayabula — APPROVED** [review]
Exceptional configuration, I have tested your configuration it works fine.

**Review by lindseyme — APPROVED** [review]
Good work

**Review by barema4 — APPROVED** [review]

**Comment by ongebo** [comment]
I cloned and tried out the branch locally, looks all good to me.

**Review by ongebo — APPROVED** [review]

File: authors/settings.py [hunk 1]
```diff
@@ -80,8 +80,12 @@
 
 DATABASES = {
```
Comment (inno-asiimwe) [inline authors/settings.py:82]: All the values are hard coded here, that is a security risk, instead use environment variables.

**Comment by inno-asiimwe** [comment]
@sekayasin please take care of the different development environments. Current configuration only takes care of a  local  environment.

File: authors/settings.py (hunk 1 above)
Comment (sekayasin) [reply authors/settings.py:82]: @inno-asiimwe that's now covered with environments variables, new changes to be pushed.

**Pull request merged**